node_modules/oae-core/manageaccess/css/manageaccess.css

```diff
@@ -186,6 +186,10 @@
         position: relative;
         top: 5px;
     }
+
+    #manageaccess-modal #manageaccess-overview-selected-invitations-container > li .oae-listitem i.fa.fa-envelope {
+        line-height: normal;
```

Comment:
I verified the vertical centering on IE9 and it was fine. I'll check 10 and 11 as well.